src/main/webapp/app/exercises/modeling/manage/example-modeling/example-modeling-submission.component.ts

```diff
@@ -379,22 +407,43 @@ export class ExampleModelingSubmissionComponent implements OnInit {
 
         this.tutorParticipationService.assessExampleSubmission(exampleSubmission, this.exerciseId).subscribe(
             () => {
+                this.markAllFeedbackToCorrect();
                 this.jhiAlertService.success('artemisApp.exampleSubmission.assessScore.success');
             },
             (error: HttpErrorResponse) => {
                 const errorType = error.headers.get('x-artemisapp-error');
 
-                if (errorType === 'error.tooLow') {
-                    this.jhiAlertService.error('artemisApp.exampleSubmission.assessScore.tooLow');
-                } else if (errorType === 'error.tooHigh') {
-                    this.jhiAlertService.error('artemisApp.exampleSubmission.assessScore.tooHigh');
+                if (errorType === 'error.invalid_assessment') {
+                    this.markAllFeedbackToCorrect();
+
+                    // Mark all wrongly made feedbacks accordingly.
+                    const correctionErrors: FeedbackCorrectionError[] = JSON.parse(error['error']['title'])['errors'];
+                    correctionErrors.forEach((res) => {
```

Comment:
why is it called `res`, I would assume, that it is a result of an http request. I would call it `correctionError`.

Reply:
Renamed it in the extracted class as mentioned.